README.md

```diff
@@ -1 +1,28 @@
-# Mobile-Menu
\ No newline at end of file
+
+# Project Name:portfolio
+This is my first project at Microverse.
+- [link is here](https://haftamudesta.github.io/desta451616-hotmail.com.github.io/portfolio.html)
+## Built With
+- HTML
+- CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+- Text editor,
+- browser like crome.
+> If you have the above mentioned  prerequistes you can download the project to your local machine from text editor by using the following steps:
+ - Clone Git repository: URL (https://github.com/haftamudesta/portfolio) 
+ - Open in your local machine
+## Author
+ - [Githab](https://github.com/settings/profile)
+- [linkedin](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+- [tiwtter](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumbs up if you like this project.
+## Acknowledgments
+Microverse
+## License
+This project is Apache License licensed.
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** To help you how to complete your README file professionally, I suggest you use this template and complete it according to your project's information.